src/app/pages/checkout-payment/payment-payone-creditcard/payment-payone-creditcard.component.html

```diff
@@ -0,0 +1,65 @@
+<div [formGroup]="payoneCreditCardForm">
+  <p class="text-danger col-sm-offset-4" id="error"></p>
```

Comment:
is this line of code used? shouldn't it be an ngIf condition here? If it is used, write a comment above, pls

Reply:
This filed is used by Payone scripts for error handling. I have added a comment to specify that.